```html
<script>
        // 失败的回调1 1
        // 成功回调4 undefined
        
        new Promise((resolve,reject)=>{
            reject(1);
        }).then(
            value =>{
                console.log("成功回调1",value);
            }
        ).then(
            value =>{
                console.log("成功回调2",value);
            }
        ).then(
            value =>{
                console.log("成功回调3",value);
            }
        ).catch(
            reason =>{
                console.log("失败的回调1",reason);
            }
        ).then(
            value =>{
                console.log("成功回调4",value);
            },
            reason =>{
                console.log("失败的回调2",reason);
            }
        )
    </script>
```